scripts/pi-hole/php/savesettings.php

```diff
@@ -452,6 +452,21 @@ function readAdlists()
 				{
 					exec('sudo pihole -a -c');
 				}
+        $adminemail = trim($_POST["adminemail"])
+        // If we didn't get an email, don't try to save it
+        if(strlen($adminemail) == 0 || !isset($adminemail))
+        {
+          $adminemail = 'noadminemail'
+        }
+        // Make sure that any email we did get is valid, otherwise error
+        elseif (!filter_var($adminemail, FILTER_VALIDATE_EMAIL)) {
+          $error  .= "Administrator email address (".htmlspecialchars($adminemail).") is invalid!<br>";
+        }
+        // Save the email after we've validated it
+        else
```

Comment:
Indentation is broken (space vs tab)

Reply:
Can you be more specific about which style is being requested?

Reply:
The code uses tabs, but this section uses spaces, which causes the indentation to be messed up.

Reply:
I can't replicate this statement @Mcat12 — when I say to Sublime change from spaces to tabs, I get a much larger diff saying the codebase is not using tabs as you said. This holds true whether I have it set to tab width of 1 or tab width of 4, as shown in the image and below.

<img width="1276" alt="screen shot 2018-05-26 at 3 23 54 pm" src="https://user-images.githubusercontent.com/947110/40579581-db02566a-60f8-11e8-92ac-e2d33e22c3fa.png">

```
~/D/g/p/AdminLTE git:fix-579 ❯❯❯ git diff                                                                        ⏎ ✱
diff --git a/settings.php b/settings.php
index 387e751d..a1b8cdbb 100644
--- a/settings.php
+++ b/settings.php
@@ -21,100 +21,100 @@ else

 ?>
 <style type="text/css">
-    .tooltip-inner {
-        max-width: none;
-        white-space: nowrap;
-    }
+       .tooltip-inner {
+               max-width: none;
+               white-space: nowrap;
+       }

-    @-webkit-keyframes Pulse {
-        from {
-            color: #630030;
-            -webkit-text-shadow: 0 0 9px #333;
-        }
-        50% {
-            color: #e33100;
-            -webkit-text-shadow: 0 0 18px #e33100;
-        }
-        to {
-            color: #630030;
-            -webkit-text-shadow: 0 0 9px #333;
-        }
-    }
+       @-webkit-keyframes Pulse {
+               from {
+                       color: #630030;
+                       -webkit-text-shadow: 0 0 9px #333;
+               }
+               50% {
+                       color: #e33100;
+                       -webkit-text-shadow: 0 0 18px #e33100;
+               }
+               to {
+                       color: #630030;
+                       -webkit-text-shadow: 0 0 9px #333;
+               }
+       }
...goes on for the entire file...
```

Reply:
🤕 this was my bad, I saw the wrong file name when I saw this. Trying again in sublime instead of Atom this time, so hope this will be better. Sorry about the extra overhead of this mess up and the rebase failure on my side.